mesonbuild/templates/cpptemplates.py

```diff
@@ -148,10 +144,6 @@ class {utoken}_PUBLIC {class_name} {{
 
 
 class CppProject(SampleImpl):
-    def __init__(self, options: Arguments):
-        super().__init__()
-        self.name = options.name
-        self.version = options.version
```

Comment:
Can you move this initializer change up one commit?

Reply:
Yup, if you do that, most of the previous diff goes away.